src/meta/txth.c

```diff
@@ -39,6 +39,7 @@ typedef enum {
     TGC = 29,           /* Tiger Game.com 4-bit ADPCM */
     ASF = 30,           /* Argonaut ASF 4-bit ADPCM */
     EAXA = 31,          /* Electronic Arts EA-XA 4-bit ADPCM v1 */
+    OKI4S = 32,         /* OKI ADPCM with 16-bit output (unlike OKI/VOX/Dialogic ADPCM's 12-bit) */
```

Comment:
Doc mismatch with TXTH.

Reply:
Yeah, I *just* fixed that.

Reply:
Oh, oops, thanks. I forgot to fix that comment when I added it to the doc; it was initially a quick tweak to see if it even worked.